### PR TITLE
false, logname, time, unclutter, view: add Persian translation

### PR DESCRIPTION
**Pull request by opoet777**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

**Comment by opoet777** [comment]
@navarroaxel Thanks for unclutter fixes.

File: pages.fa/common/time.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# time
+
+> نمایش زمان اجرای یک دستور.
+
+- نمایش زمان اجرای دستور "ls":
```
Comment (bl-ue) [inline pages.fa/common/time.md:5]: The English version was changed in a WIP PR:
```suggestion
- نمایش زمان اجرای دستور `ls`:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
LGTM, except one thing was changed in the English version of `time.md` in a [WIP PR](https://github.com/tldr-pages/tldr/pull/5119).

**Review by bl-ue — APPROVED** [review]
Thank you @opoet777! :heart:

**Pull request merged**